<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Helmerttransformation_gui</class>
 <widget class="QDialog" name="Helmerttransformation_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helmerttransformation</string>
  </property>
  <widget class="QPushButton" name="pushButton_datei">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>json-Datei laden</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dateipfad">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_berechne">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>berechne</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;Transformationsparamenter:&lt;/b&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ma√üstab</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Drehwinkel (gon)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Translation Y&lt;sub&gt;0&lt;/sub&gt; (m)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_m">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_winkel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_transY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_a">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_o">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_test">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Testdatei 1</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_in">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>49</y>
     <width>261</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_out">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>261</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Translation X&lt;sub&gt;0&lt;/sub&gt; (m)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_transX">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_test2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Testdatei 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_speichern">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>tansformierte Punkte als JSON - Datei speichern</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_helmert">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;Eingangsdatei&lt;/b&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;transformierte Punkte&lt;/b&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>20</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Restklaffen identischer Punkte&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_rstklf">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>50</y>
     <width>261</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
